pycontw/pycon.tw
# Add a tick box to the proposal form indicating 1st-time speaker

## Add a tick box to the proposal form indicating 1st-time speaker

## Description
Support program team's plan of young talent. 
Add a tick box in talk proposal & tutorial proposal form for users indicating whether she/he is the first time speaker.

## Steps to Test This Pull Request
Steps to reproduce the behavior:
1. Go to 'MY PYCON'
2. Click on 'proposals' tab
4. Click on '+New Talk Proposal' or '+New Tutorial Proposal''
3. Scroll down
4. See tick box of *First time speaker*

## Expected behavior
After the proposal form is submitted, the newly added data in the talk proposal and toturial proposal tables will contain information about whether this user is the first time speaker.

## More Information
**Screenshots**
- Frontend

![image](https://user-images.githubusercontent.com/20749936/106355671-7ce89a80-6334-11eb-98e4-58ec4c685f22.png)

- Django Administration
![image](https://user-images.githubusercontent.com/20749936/106355783-2c257180-6335-11eb-9261-827ca0531c2c.png)
![image](https://user-images.githubusercontent.com/20749936/106355705-b9b49180-6334-11eb-9a60-5aecf18c422a.png)
